fonts.css

```diff
@@ -0,0 +1,4 @@
+@font-face {
+    font-family: "Montserrat";
+    src: url("//fonts.fontstorage.com/import/montserrat.css") format("woff2");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/eyurodin_axion/pullRequest?prid=5699758)